test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -0,0 +1,67 @@
+load "${REPOSITORY_ROOT}/test/helper/setup"
+load "${REPOSITORY_ROOT}/test/helper/common"
+
+TEST_NAME_PREFIX='Spam junk folder:'
+
+# Test case
+# ---------
+# When SPAMASSASSIN_SPAM_TO_INBOX=1, spam messages must be delivered
+# and eventually (MOVE_SPAM_TO_JUNK=1) moved to the Junk folder.
+
+@test "${TEST_NAME_PREFIX} (Amavis) spam message delivered & moved to Junk folder" {
+  CONTAINER_NAME='dms-test-spam_junk_folder-1'
```

Comment:
What exactly should? Probably this is related to <https://github.com/docker-mailserver/docker-mailserver/pull/2938/files#r1056041092> and hence, somewhat resolved?

Reply:
> What exactly should?

`CONTAINER_NAME` in most of your converted tests is defined statically (_or variants are `CONTAINER_NAME1`, `CONTAINER_NAME2`, and re-assigned to `CONTAINER_NAME` in a test case_) at the top right after `TEST_NAME_PREFIX` is defined.

---

I was pointing out that in this file you seemed to have been inconsistent with that convention. The referenced feedback you asked about is hand-waving the concern about `CONTAINER_NAME` usage for assertions instead of `CONTAINER_NAME1` or similar as all the assertions are within the same test case definition, so I'm not as fussed as I was when I initially read them thinking there was more than 2 test cases present.

I am not blocking on this, just an observation that appears to be breaking convention for convenience.